addons/mail/static/src/core/common/suggestion_hook.js

```diff
@@ -173,19 +177,23 @@ class UseSuggestion {
         if (!this.search.delimiter) {
             return;
         }
-        const { type, suggestions } = this.suggestionService.searchSuggestions(this.search, {
+        const {
+            type,
+            suggestions,
+            specialSuggestions = [],
+        } = this.suggestionService.searchSuggestions(this.search, {
```

Comment:
You could put `specialSuggestions` inside `suggestions`, so that you keep same shape of the returned value of `searchSuggestions`